dataland-frontend/tests/component/components/resources/frameworkDataSearch/gdvVoeb/GdvVoebPanel.cy.ts

```diff
@@ -0,0 +1,153 @@
+import { type FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
+import { type CompanyAssociatedDataGdvData, DataTypeEnum, type GdvData } from "@clients/backend";
+
+import { mountMLDTFrameworkPanelFromFakeFixture } from "@ct/testUtils/MultiLayerDataTableComponentTestUtils";
+import { type MLDTConfig } from "@/components/resources/dataTable/MultiLayerDataTableConfiguration";
+
+import { type AvailableMLDTDisplayObjectTypes } from "@/components/resources/dataTable/MultiLayerDataTableCellDisplayer";
+import { formatGdvYearlyDecimalTimeseriesDataForTable } from "@/components/resources/dataTable/conversion/gdv/GdvYearlyDecimalTimeseriesDataGetterFactory";
+import {
+  formatStringForDatatable
+} from "@/components/resources/dataTable/conversion/PlainStringValueGetterFactory";
+import {
+  formatListOfBaseDataPoint
+} from "@/components/resources/dataTable/conversion/gdv/GdvListOfBaseDataPointGetterFactory";
+import {formatNumberToReadableFormat} from "../../../../../../src/utils/Formatter";
+
+const configForGdvVoebPanelWithOneRollingWindow: MLDTConfig<GdvData> = [
+  {
+    type: "cell",
+    label: "Überwachung der Einkommensungleichheit",
```

Comment:
Why are those labels needed?

Reply:
Because this is a view config, and a view config expects a field "label" for each config-element.  It is expected by the typescript compiler per type definition, therefore it is required here.

But another thing I just noticed:   The name of this file here violates our component test naming convention and seems pretty inconsistent to other view page test files. We normally name our component tests according to the vue component that they are testing, or - if not possible as for most of the view page tests - we name it smth like "MultiLayerDataTableWithSfdr.cy.ts" or "MultiLayerDataTableWithLksg.cy.ts".
The current name is misleading because there is no vue component "GdvVoebPanel."

Reply:
Good point regarding the test name

Reply:
I will close this, as the name was refactored and the file moved. the other comment in regards to this file i leave open until the test is redesigned